### ISSUE TITLE
Fix tag versioning here and on NPM

### ISSUE THREAD

**Issue by thiagofesta** [Glavin001/ember-jsoneditor#12]
You released v1.0.0 here, but on NPM it is called v0.1.2.

It should be ideal to always keep same versioning numbers.

**Comment by Glavin001** [comment]
Hah, nice eye. I had 1 un-pushed commit and that commit was changing the version number.
https://github.com/Glavin001/ember-jsoneditor/commit/c507686107e1da1063f883d5989d96b044be2611
Thanks!

**Comment by thiagofesta** [comment]
Sure, don't forget to `npm publish`

**Comment by Glavin001** [comment]
Yup, it was already published to npm for v0.1.2: https://www.npmjs.com/package/ember-jsoneditor . The problem was that I didn't push to GitHub the version bump.

**Comment by thiagofesta** [comment]
@Glavin001 yeah, that was one of the issues as well... but I see this: https://github.com/Glavin001/ember-jsoneditor/tree/v1.0.0 on github tags, but not on NPM

**Comment by Glavin001** [comment]
The GitHub tags aren't being used nor are up-to-date. Looks like I made the v1.0.0 tag back in https://github.com/Glavin001/ember-jsoneditor/pull/2#issuecomment-72378339 for Ember-CLI users along with the v0.1.0 (correct) tag for non-Ember-CLI users. Unfortunately, the actually version of the next release ended up being v0.1.2 instead.
I've deleted the incorrectly labelled tag v1.0.0 and added tag v0.1.2 which should reflect what is also in NPM.
